static/gsApp/components/ai/aiSetupConfiguration.tsx

```diff
@@ -0,0 +1,134 @@
+      <Stack align="center">
+        <MeetSeerPanel>
+          <Stack gap="2xl" padding="2xl">
+            <Heading as="h2" size="2xl" align="center">
+              {t('Meet Seer')}
+            </Heading>
+            <Text>
+              {t(
+                'Debug faster with Seer. It will connect to your repositories, scan all of your issues, highlight the ones that are quick to fix, and propose solutions. You can even integrate with your favorite coding agent to implement changes in code. '
+              )}
+            </Text>
+            <Stack align="center">
+              {hasBillingAccess(organization) ? (
+                <LinkButton
+                  to="/settings/billing/overview/?product=seer"
+                  priority="primary"
+                  icon={<IconUpgrade />}
+                >
+                  {t('Try Out Seer Now')}
+                </LinkButton>
+              ) : (
+                <Alert type="warning">
+                  {t(
+                    'You need to be a billing member to try out Seer. Please contact your organization owner to upgrade your plan.'
+                  )}
+                </Alert>
+              )}
+            </Stack>
+          </Stack>
+        </MeetSeerPanel>
+        <Stack width="70%" align="center">
+          <ImageContainer width="250px" height="120px">
+            <Image alignSelf="flex-start" src={seerAutofixImg} />
+          </ImageContainer>
+          <SeerFeaturesPanel width="100%">
+            <Stack direction="row" gap="md" padding="md">
+              <ImageContainer aspectRatio="16 / 9" minWidth="30px" width="10%">
+                <Image src={seerConfigConnectImg} />
+              </ImageContainer>
+              <Stack gap="sm" padding="sm">
+                <Heading as="h3">{t('Root Cause Analysis & Code Fixes')}</Heading>
+                <Text>
+                  {t(
+                    'Seer analyzes the root cause of an issue and propose fixes ready to merge as draft PRs.'
+                  )}
+                </Text>
+              </Stack>
+            </Stack>
+          </SeerFeaturesPanel>
+          <SeerFeaturesPanel width="100%">
+            <Stack direction="row" gap="md" padding="md">
+              <ImageContainer aspectRatio="16 / 9" minWidth="30px" width="10%">
+                <Image src={seerConfigCheckImg} />
+              </ImageContainer>
+              <Stack gap="sm" padding="sm">
+                <Heading as="h3">{t('AI Code Review')}</Heading>
+                <Text>{t('Seer catches bugs in your PRs before you ship them.')}</Text>
+              </Stack>
+            </Stack>
+          </SeerFeaturesPanel>
+        </Stack>
+      </Stack>
+    </Fragment>
+  );
+}
+
+const HeroImage = styled('img')`
+  position: absolute;
+  z-index: ${p => p.theme.zIndex.initial};
+  min-width: 150%;
+  left: 50%;
+  transform: translateX(-47%) translateY(-35%);
+`;
+
+const MeetSeerPanel = styled(Panel)`
+  margin-top: 32%;
```

Comment:
Will these % based sizes will look good when the slideout panel is resized?

Looks like it can be wider/narrower with a click+drag

Reply:
I tested this locally and it looks decent at various sizes with the current values.